diff --git a/node_modules/react-native-html-to-pdf/android/src/main/java/android/print/PdfConverter.java b/node_modules/react-native-html-to-pdf/android/src/main/java/android/print/PdfConverter.java
index 38f13ef..a55c758 100644
--- a/node_modules/react-native-html-to-pdf/android/src/main/java/android/print/PdfConverter.java
+++ b/node_modules/react-native-html-to-pdf/android/src/main/java/android/print/PdfConverter.java
@@ -96,7 +96,7 @@ public class PdfConverter implements Runnable {
         });
         WebSettings settings = mWebView.getSettings();
         settings.setDefaultTextEncodingName("utf-8");
-        mWebView.loadData(mHtmlString, "text/HTML; charset=utf-8", null);
+        mWebView.loadDataWithBaseURL(mBaseURL, mHtmlString, "text/HTML", "utf-8", null);
     }
 
     public PrintAttributes getPdfPrintAttrs() {
